db/models/feedback.js

```diff
@@ -39,10 +39,27 @@ class Feedback extends Model {
             })
         ]);
     }
-    static findAllByDescription() {
-        return this.findAll({
+    static findAllByDescription(description = null) {
```

Comment:
I'd probably make this two methods, `findAllByDescription` which is the previous behaviour and `findByDescription` which is this new finder.

Reply:
Maybe even rename `findAllByDescription` to `findAllGroupedByDescription` to make it  clear it's grouping results.

Reply:
👍 done